### PR TITLE
21802 - Add details call and list.

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:21802 *https://github.com/bcgov/entity/issues/21802 

*Description of changes:*
- add details button
- comments visible
- prepared code for putting in popup
- tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-21-w2akg042.web.app

File: src/components/bcros/filing/CommonTemplate.vue [hunk 1]
```diff
@@ -17,19 +17,19 @@
...
-            color="primary"
-            :ripple="false"
-            @click.stop="isShowBody = !isShowBody"
-          >
-            <UIcon name="i-mdi-message-reply" size="small" />
-            <span>
-              {{ isShowBody ? $t('label.filing.detail') : $t('label.filing.detail') }}
-              ({{ filing.commentsCount }})</span>
-          </UButton>
+          <diV>
```
Comment (kialj876) [inline src/components/bcros/filing/CommonTemplate.vue:20]: capital 'V' here?

**Review by kialj876 — COMMENTED** [review]
Hey this is looking really good, just needs CI fix and its good to go I think

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-21-w2akg042.web.app

**Review by kialj876 — APPROVED** [review]
Looks good!

**Pull request merged**